langs.json

```diff
@@ -944,6 +944,11 @@
         "two": "iu",
         "three": "iku"
     },
+    {
+        "names": ["Ila - Zambia"],
+        "two": "",
+        "three": "ila"
```

Comment:
i think this should be "ilb".
When i added languages earlier i checked what code for them is being followed.

"Language codes" at https://en.wikipedia.org/wiki/Ila_language.
We were doing integration with a third party and it became relevant that HQ used the same code as used by the standards.

Reply:
updated